.ci/Jenkinsfile

```diff
@@ -220,6 +220,66 @@ pipeline {
         }
       }
     }
+    stage('Stack Release Tag') {
+      options {
+        skipDefaultCheckout()
+      }
+      when {
+        beforeAgent true
+        allOf {
+          branch 'main'
+          expression { return params.release }
+        }
+      }
+      agent { label 'ubuntu-18 && immutable' }
+      stages {
+        stage('Notify') {
+          options { skipDefaultCheckout() }
+          steps {
+            notifyStatus(slackStatus: 'warning', subject: "[${env.REPO}] Stack Release ready to be pushed",
+                         body: "Please go to (<${env.BUILD_URL}input|here>) to approve or reject within 12 hours.")
+          }
+        }
+        stage('Stack Release CI') {
+          options { skipDefaultCheckout() }
+          input {
+            message 'Should we tag the current version as a stack release?'
+            ok 'Yes, we should.'
+          }
+          environment {
+            NPM_PACKAGE = '@elastic/synthetics'
+          }
+          steps {
+            cleanup()
+            withNodeJSEnv() {
+              dir("${BASE_DIR}") {
+                withNpmrc(secret: "${env.NPMRC_SECRET}", path: "${env.WORKSPACE}/${env.BASE_DIR}") {
+                  withTotpVault(secret: "${env.TOTP_SECRET}", code_var_name: 'TOTP_CODE') {
+                    setEnvVar('VERSION', sh(script: 'npm show . version', label: 'tag the version as stack release', returnStdout: true).trim())
+                    sh(script: "npm dist-tag --otp=${env.TOTP_CODE} add ${env.NPM_PACKAGE}@${env.VERSION} stack_release", label: 'tag stack_release')
+                  }
+                }
+              }
+            }
+          }
+          post {
+            success {
+              notifyStatus(slackStatus: 'good', subject: "[${env.REPO}] Stack Release '${env.VERSION}' published", body: "Great news, the stack release has been done successfully. (<${env.RUN_DISPLAY_URL}|Open>). \n Release URL: (<https://preview.npmjs.com/package/${env.NPM_PACKAGE}/v/${env.VERSION}|${env.VERSION}>)")
```

Comment:
This URL matches with the one in L207, is the same URL? Or a different one pointing to `stack_release`?

Reply:
It should be the same URL since there is only one page where we can find the tag and the version.
For example: https://www.npmjs.com/package/@elastic/synthetics/v/1.0.0-beta.31